README.md

```diff
@@ -27,15 +27,15 @@ below) and is faster, too.
 Create a runtime profile with
 
 ```
-python -mcProfile -o program.prof yourfile.py
+python -m cProfile -o program.prof yourfile.py
 ```
 
 or an [import
 profile](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPROFILEIMPORTTIME)
 with
 
 ```
-python -X importtime yourfile.py 2> import.log
+python -X import time yourfile.py 2> import.log
```

Comment:
`import time` is incorrect, it should be `importtime`: https://docs.python.org/3/using/cmdline.html#cmdoption-X

```suggestion
python -X importtime yourfile.py 2> import.log
```